app/views/videos/show.html.haml

```diff
@@ -7,6 +7,37 @@
         %header
           %h3= @video.title
           %h4=link_to @video.category.name, category_path(@video.category)
-          %span Rating: 4.5/5.0
+          %span
+            - if @video.average_rating != nil
+              = "Rating: #{@video.average_rating}/5.0"
+            - else
+              = "There are no reviews"
         %p= @video.description
         .actions
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for([@video, Review.new]) do |f|
+        = f.form_group do
+          %fieldset
+            .row
+              .col-sm-3
+                = f.select :rating, [["5 Stars", 5], ["4 Stars", 4], ["3 Stars", 3], ["2 Stars", 2], ["1 Star", 1]], { label: "Rate this video" }
```

Comment:
`[["5 Stars", 5], ["4 Stars", 4], ["3 Stars", 3], ["2 Stars", 2], ["1 Star", 1]]` this part is hardcoded - try to think about some algorithm that builds it. Use helper method to place your method.